public/js/main.js

```diff
@@ -216,24 +217,45 @@ $(document).ready(() => {
     const B = $('#tradeCurrency').val();
     console.log(`Loading chart for ${A}_${B}...`);
     $.getJSON(`/data?currencyA=${A}&currencyB=${B}`, (data) => {
+      console.log(data);
       const ops = {};
       ops.A = A; ops.B = B; ops.data = data; chartData = data;
-      $('#hcharts-strategy').highcharts('StockChart', createHighChartsObj(ops));
+      mainChart = Highcharts.stockChart('hcharts-strategy', createHighChartsObj(ops));
+      //$('#hcharts-strategy').highcharts('StockChart', createHighChartsObj(ops));
     });
   });
   $('#addIndicatorBtn_SMA').click(() => {
     console.log('### Inside addIndicator_SMA');
     const SMAParam = $('#addIndicator_SMA').val();
+
+    // Parsing pattern for csv SMA values
+    var pattern = /\s*,\s*/;
+    var nameList = SMAParam.split(pattern);
+
+    // Looping through each entered value and adding a SMA series
+    for (let i=0; i < nameList.length; i++){
+      var SMA = window.SMA(chartData, nameList[i]);
+      // Creating a unique id for this indicator
+      var id = "SMA - "+ nameList[i]
+
+      // Creating indicator series for selected indicator
+      createIndicatorSeries(mainChart, id, true);
+      const targetSeries = mainChart.get(id);
+
+      // Adding indicator series to the chart
+      addTickerDatasetToSeries(targetSeries, SMA);
+    }
+
     console.log(SMAParam);
-    console.log(chartData[0]);
-    const SMA = window.SMA(chartData, SMAParam);
-    console.log(SMA);
-
-    const newIndObj = createIndicatorSeries('SMA', SMA, true);
-    console.log(newIndObj);
-    console.log(mainChart);
-    mainChart.addSeries(newIndObj);
-    
+    console.log(chartData);
+
+    // Redrawing chart after adding indicators
+    mainChart.redraw();
+
+    //console.log(newIndObj);
```

Comment:
delete commented out things that we probably won't use again

Reply:
I didn't want to delete your code

Reply:
if you're replacing a feature/implementation, you can delete the code associated with that feature, unless we'll need it in the future.